vespa-http-client/src/main/java/com/yahoo/vespa/http/client/config/FeedParams.java

```diff
@@ -57,7 +54,7 @@ public long getMaxSleepTimeMs() {
          * @return this, for chaining
          */
         @Beta
-        public Builder withSilentUpgrade(boolean silentUpgrade) {
+        public Builder setSilentUpgrade(boolean silentUpgrade) {
```

Comment:
Is this considered a public API and thus may break something?

Reply:
Yes, but this is marked @Beta so it's ok.